docs/release_notes/v1.14.1.adoc

```diff
@@ -0,0 +1,39 @@
+[id="{p}-release-notes-v1.14.1"]
+=== Elastic Cloud Control (ecctl) v1.14.1
+++++
+<titleabbrev>v1.14.1</titleabbrev>
+++++
+
+<<{p}-release-notes-v1.14.1-whats-new,What's new>> | <<{p}-release-notes-v1.14.1-bug-fixes,Bug fixes>> | <<{p}-release-notes-v1.14.1-breaking-changes,Breaking changes>> | <<{p}-release-notes-v1.14.1-changelog,Changelog>>
```

Comment:
You may get some errors because these links reference some non-existent sections. So these also have to be manually cleaned up.

Reply:
Great point, thanks! 🙇‍♀️ Fixed in https://github.com/elastic/ecctl/pull/672/commits/c4b6d7f620d17f889d5ed7a882fdab74076c4025.